acq4/devices/PatchPipette/states.py

```diff
@@ -1265,7 +1267,7 @@ def run(self):
         dev.clean = True
         self.resetPosition()
         dev.newPatchAttempt()
-        return 'out'          
+        return config['fallbackState']
```

Comment:
why?

Reply:
fallbackState is 'out', and isn't used ever, so I had just imagined they were meant to go together.

Reply:
"isn't used ever"... in this class. the base class sets `self.nextState` to it, so it is still in use there.